### DISCUSSION TITLE
How to view data over longer/slower time interval?

### DISCUSSION THREAD

**Discussion by n-flock** [facontidavide/PlotJuggler#556] [Q&A]
I am working with data logs that sample at 10Hz over typically 75-120 minutes.  Using time stamp as x-axis values in plots shows microseconds on x-axis.  Would like to view data in seconds or minutes.  Not finding a way to do this in Plotjuggler.  Could manipulate the data in the CSV file but hoping to not have to do that.

**Comment by facontidavide** [comment]
The X axis should be already in seconds, unless you use DataTime option. 

Maybe it would be easier to look at your specific data, to understand what is going on.

There is **no way to change the unit to minutes.**

**Reply by n-flock** [reply]
Data set attached below. 

[Motor_Log_211029.csv](https://github.com/facontidavide/PlotJuggler/files/7458003/Motor_Log_211029.csv)

Time is recorded at 10 Hz and stored as µs.  Is there a way to scale the x-axis (by 10e5 in this case)?  Here is what I am seeing in PJ:

![image](https://user-images.githubusercontent.com/93461195/139789934-2de2fe63-18dc-4b14-adba-8c4a74f45798.png)

Is is possible to average every 10 data points into full seconds?
